ui/tests/unit/filter-simple-condition-widget.spec.ts

```diff
@@ -539,28 +440,8 @@ describe('Widget FilterSimpleCondition', () => {
       });
     };
 
-    it('should transform an invalid "from" operator when relative date are disabled', async () => {
-      createWrapper('from');
-      await wrapper.vm.$nextTick();
-      expect(wrapper.emitted().input).toHaveLength(1);
-      // should emit new value with 'ge' operator replacing invalid 'from' operator
-      expect(wrapper.emitted().input[0]).toEqual([
-        { column: 'columnA', value: '', operator: 'ge' },
-      ]);
-    });
-
-    it('should transform an invalid "until" operator when relative date are disabled', async () => {
-      createWrapper('until');
-      await wrapper.vm.$nextTick();
-      expect(wrapper.emitted().input).toHaveLength(1);
-      // should emit new value with 'le' operator replacing invalid 'until' operator
-      expect(wrapper.emitted().input[0]).toEqual([
-        { column: 'columnA', value: '', operator: 'le' },
-      ]);
-    });
-
     it('should transform an invalid "le" operator when relative date are enabled', async () => {
```

Comment:
Maybe we can remove the "when relative date are enabled" from test titles ?

Reply:
Yes you're right ! Fixed in [44bb611](https://github.com/ToucanToco/weaverbird/pull/1900/commits/44bb61154b74948aef85831742a9b376b2d83711) 🙏